tests/emukit/bayesian_optimization/test_q_expected_improvement.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
```

Comment:
That's great to have some tests. We also have generic file to test acquisition functions. Would you be able to add QEI there too?

https://github.com/amzn/emukit/blob/master/tests/emukit/test_acquisitions.py

Reply:
Oh, I didn't notice this before as it wasn't in the bayesian_optimization-folder. All acquisition functions in the source code are in the bayesian_optimization subfolder. Would it make sense to move this test to the correct folder?

Reply:
this test is there because you can use it with any acquisition function, not only those used with BO. let's keep it where it is

Reply:
After having a look, the test set "test:acquisitions" isn't suitable for testing acquisition functions that evaluate a single value for a batch of points. I'll keep the tests as they are.